src/webview/leetCodePreviewProvider.ts

```diff
@@ -14,22 +16,29 @@ class LeetCodePreviewProvider implements Disposable {
         this.context = context;
     }
 
-    public async preview(node: IProblem): Promise<void> {
+    public async show(node: IProblem): Promise<void> {
+        // Fetch problem first before creating webview panel
+        const descString: string = await leetCodeExecutor.getDescription(node);
+
         this.node = node;
         if (!this.panel) {
-            this.panel = window.createWebviewPanel("leetcode.preview", "Preview Problem", ViewColumn.Active, {
+            this.panel = window.createWebviewPanel("leetcode.preview", "Preview Problem", ViewColumn.One, {
                 enableScripts: true,
                 enableCommandUris: true,
                 enableFindWidget: true,
                 retainContextWhenHidden: true,
+                localResourceRoots: markdownEngine.localResourceRoots,
             });
 
             this.panel.webview.onDidReceiveMessage(async (message: IWebViewMessage) => {
                 switch (message.command) {
-                    case "ShowProblem":
+                    case "ShowProblem": {
                         await commands.executeCommand("leetcode.showProblem", this.node);
-                        this.dispose();
-                        return;
+                        await commands.executeCommand("workbench.action.focusSecondEditorGroup");
```

Comment:
This line relates to (maybe) a bug in vscode, and I have reported in a issue:
https://github.com/Microsoft/vscode/issues/71608, which makes me have to create column 2 first before moving (by calling `reveal(ViewColumn.Two)`) the webview panel to it.

I've successfully build the vscode repository locally in the aftertoon, and I'd like to have a try to debug the vscode myself.